src/alph_puzzle.c

```diff
@@ -195,10 +195,10 @@ static void Task_AlphPuzzle_WaitDropCursorAnimOnQuit(SysTask *task, void *_data)
 
 // Used to detect hovering over corners
 static const u8 sCornerCoords[4][2] = {
-    { 0, 0 },
-    { 5, 0 },
-    { 0, 5 },
-    { 5, 5 },
+    {0,  0},
```

Comment:
clang-format, go home, you're drunk. and this is coming from someone who is drunk and ain't sobering up anytime soon.